package.json

```diff
@@ -16,20 +16,20 @@
   "scripts": {
     "build": "stencil build",
     "docs": "stencil build --config stencil.docs.config.ts",
+    "deps:update": "updtr --ex @types/jest @types/puppeteer jest jest-cli puppeteer && git add package*.json && git commit -q -m 'chore(deps): Bump versions.'",
     "lint": "tslint src/components/**/*.ts?(x)",
     "start": "stencil build --dev --watch --serve",
     "test": "stencil test --spec --e2e",
     "test:headed": "stencil test --spec --e2e --devtools",
     "test:watch": "stencil test --spec --e2e --watch",
     "test:watchAll": "stencil test --spec --e2e --watchAll",
-    "preversion": "npm ci && npm test",
-    "version": "npm run build && npm run docs && git add .",
-    "postversion": "git push && git push --tags",
     "prepublishOnly": "node ./support/prepublish.js",
-    "release:patch": "npm version patch && npm publish",
-    "release:minor": "npm version minor && npm publish",
-    "release:major": "npm version major && npm publish",
-    "deps:update": "updtr --ex @types/jest @types/puppeteer jest jest-cli puppeteer && git add package*.json && git commit -q -m 'chore(deps): Bump versions.'"
+    "prepare:patch": "npm version patch",
+    "prepare:minor": "npm version minor",
+    "prepare:major": "npm version major",
```

Comment:
I don't think we need these scripts. We can just run the standard commands these alias.

Reply:
Are you talking about `preversion` and `postversion` scripts?
running npm version <foo> should still run the `preversion` and `postversion` scripts.

Reply:
Ah, I misread your comment. Were you suggesting dropping these and just running `npm version <major | minor | patch>`?